cdc/sorter/unified/backend_pool.go

```diff
@@ -99,6 +99,12 @@ func newBackEndPool(dir string, captureAddr string) (*backEndPool, error) {
 		metricSorterOnDiskDataSizeGauge := sorter.OnDiskDataSizeGauge.WithLabelValues(captureAddr, id)
 		metricSorterOpenFileCountGauge := sorter.OpenFileCountGauge.WithLabelValues(captureAddr, id)
 
+		// TODO: The underlaying implementation only recognizes cgroups set by
+		// containers, we need to support cgroups set by systemd or manually.
+		totalMemory, err := memory.MemTotal()
+		if err != nil {
+			log.Panic("read memory stat failed", zap.Error(err))
+		}
```

Comment:
I notice there is no unit test in `github.com/pingcap/tidb/util/memory`, can we add a unit test in github action to verify it works.

Reply:
I think its hard to add unit tests, the best effort we can do is to verify if `github.com/pingcap/tidb/util/memory` parses cgroup info correctly, and it should be added in upstream not here.

Also, we plan to support cgroups set by systemd in a later PR, at that time, we can add unit tests.